utest/contrib/testrunner/test_command_builder.py

```diff
@@ -0,0 +1,56 @@
+#  Copyright 2008-2015 Nokia Networks
+#  Copyright 2016-     Robot Framework Foundation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import unittest
+
+from robotide.contrib.testrunner.CommandBuilder import CommandBuilder
+
+
+class CommandBuilderStub(CommandBuilder):
+    def _get_listener_path(self):
+        return 'C:\\My Work\\Python\\TestRunnerAgent.py'
```

Comment:
This cannot be dependent on the operating system, at least not Windows, but POSIX.

Reply:
OK, let's keep it because is only text. I was worried if real paths.

Reply:
I changed "builder". FileWriter is static class. It's not good, I hade some problems with testing my logic and decided move FileWriter to another place. This paths for example only. To be honest, I don't know how path look like on Linux. We can add here a few additional tests for Linux.

Reply:
And this is my style to writing unit tests. I usually provide some common examples how it works. I prefer to use the most realistic values for variables and names in tests. I hope it will be usefull for people, who will try to understand how is it working.